hashintel/hash
# Implement optimistic UI for table block

## 🌟 What is the purpose of this PR?

This PR implements optimistic UI for table block to provide better UX

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203801045329780/1204298893311428/f) _(internal)_

## 🔍 What does this change?

- Cleanup css
- Move few components to their own files
- Implement optimistic UI for table block

## 🚀 Has this modified a publishable library?

This PR:


## ❓ How to test this?



1.  Checkout the branch / view the deployment
1.  Run table block
1. Set `simulateDatastoreLatency` to a high value (e.g. `1000`)
2. Check if updating table rows/columns work immediately (optimistic UI)
